libraries/debian_handler.rb

```diff
@@ -34,6 +34,9 @@ def handle_uninstall
     private
 
     def configure_package(action_name)
+      byop = !node['chef-ingredient'].nil? && \
```

Comment:
this could be simplified to `byop = node['chef-ingredient'] && node['chef-ingredient']['byop']`